src/uucore/src/lib/mods/clap_localization.rs

```diff
@@ -120,167 +105,209 @@ pub fn handle_clap_error_with_exit_code(err: Error, util_name: &str, exit_code:
-                let colored_error_word = maybe_colorize(&error_word, Color::Red);
+        // For ValueValidation errors, include the validation error details
+        // Note: We don't print these separately anymore as they're part of the main message
 
-                // Print main error message with fallback
-                let error_msg = translate!(
-                    "clap-error-unexpected-argument",
-                    "arg" => colored_arg.clone(),
-                    "error_word" => colored_error_word.clone()
-                );
-                eprintln!("{error_msg}");
+        // Show possible values if available (for InvalidValue errors)
+        if matches!(err.kind(), ErrorKind::InvalidValue) {
+            if let Some(valid_values) = err.get(ContextKind::ValidValue) {
                 eprintln!();
+                let possible_values_label = translate!("clap-error-possible-values");
+                eprintln!("  [{}: {}]", possible_values_label, valid_values);
+            }
+        }
 
-                // Show suggestion if available
-                let suggestion = err.get(ContextKind::SuggestedArg);
-                if let Some(suggested_arg) = suggestion {
-                    let tip_word = translate!("common-tip");
-                    let colored_tip_word = maybe_colorize(&tip_word, Color::Green);
-                    let colored_suggestion =
-                        maybe_colorize(&suggested_arg.to_string(), Color::Green);
-                    let suggestion_msg = translate!(
-                        "clap-error-similar-argument",
-                        "tip_word" => colored_tip_word.clone(),
-                        "suggestion" => colored_suggestion.clone()
-                    );
-                    eprintln!("{suggestion_msg}");
-                    eprintln!();
-                } else {
-                    // Look for other clap tips (like "-- --file-with-dash") that aren't suggestions
-                    // These usually start with "  tip:" and contain useful information
-                    for line in _lines.iter() {
-                        if line.trim().starts_with("tip:") && !line.contains("similar argument") {
-                            eprintln!("{}", line);
-                            eprintln!();
-                        }
-                    }
-                }
+        eprintln!();
+        eprintln!("{}", translate!("common-help-suggestion"));
+        std::process::exit(1);
+    } else {
+        // Fallback if we can't extract context - use clap's default formatting
+        let rendered_str = err.render().to_string();
+        let lines: Vec<&str> = rendered_str.lines().collect();
+        if let Some(main_error_line) = lines.first() {
+            eprintln!("{}", main_error_line);
+            eprintln!();
+            eprintln!("{}", translate!("common-help-suggestion"));
+        } else {
+            eprint!("{}", err.render());
+        }
+        std::process::exit(1);
+    }
```

Comment:
Both blocks end with `std::process::exit(1);` and so I would move it outside.
```suggestion
    } else {
        // Fallback if we can't extract context - use clap's default formatting
        let rendered_str = err.render().to_string();
        let lines: Vec<&str> = rendered_str.lines().collect();
        if let Some(main_error_line) = lines.first() {
            eprintln!("{}", main_error_line);
            eprintln!();
            eprintln!("{}", translate!("common-help-suggestion"));
        } else {
            eprint!("{}", err.render());
        }
    }
    std::process::exit(1);
```